packages/account/src/Components/article/article.tsx

```diff
@@ -10,7 +10,7 @@ type TDescriptionsItem = {
 };
 
 export type TArticle = {
-    title: string;
+    title: JSX.Element | string;
```

Comment:
Can use ReactNode as it includes both of this 😸

Reply:
ReactNode can be null or undefined as well, which is not suitable for our case 🙏🏼